### ISSUE TITLE
Configuration of Python keyword style

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/python-jsonpreprocessor#224]
In default case and if explicitly coded by `CJsonPreprocessor(syntax="python")`, this works:

```
"param1" : True,
"param2" : False,
"param3" : None
```

But switching to pure JSON format with

`CJsonPreprocessor(syntax="json")`

the above code causes

`Expecting value: line 2 column 15 (char 16)'!`

This error message is unfavorable. There is no value missing. There is a value found that is not expected or not supported.

But do we really need this "`syntax`" parameter? I don't think so.

I can do this (with still: `syntax="json"`):

`"param1" : ${True}`

And I get boolean `True`.

**Comment by namsonx** [comment]
Hello Holger,

Regarding the error message when user inputs:
```
"param1" : True,
"param2" : False,
"param3" : None
```
with ```syntax="json"```, after our error logging improvement, the error message now is:
```
Exception: Expecting value: line 2 column 12 (char 13)
Nearby: '... {\n"param1" : True,\n"param2" : Fal ...'
In file: 'C:/Users/mas2hc/Desktop/data_storage/workspace/ROBFWAIO-OSS/python-jsonpreprocessor/testing/json_test.jsonp'
```

Based on the ```syntax``` parameter, this is one of requirements we implemented before.
JSON syntax, it uses ```true```, ```false```, and ```null``` while Python uses ```True```, ```False```, and ```None```. This requirement allows users can use ```bool``` and ```NoneType``` data types in .jsonp following JSON syntax or Python syntax.

Thank you,
Son

**Comment by HolQue** [comment]
Retest successful. Issue can be closed.

**Comment by test-fullautomation** [comment]
released with RobotFramework AIO 0.13.0
